### PR TITLE
Create page for the Merge Data Challenge

### PR DESCRIPTION
**Pull request by Rodrigolvc**
## Description

Hey Everyone,
I was trying to make changes to the previous PR made by Snapcrackle (#6970), but could not figure out how.  This is my first PR; let me know if I made any mistakes or if there are improvements you could suggest. I included the hero image we want to use for the page; let me know if we need to scale/crop. 

**Preview deploy:**
https://ethereumorgwebsitedev01-rodrigolvcethereumorgwebsiteda.gtsb.io/en/community/grants/merge-data-challenge/

## Related Issue

Not related to any open issues.
Merge-Data-Challenge-hero-large

### CODE REVIEW

File: src/content/community/grants/merge-data-challenge/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+_Up until four weeks after the Merge, the Ethereum Foundation is sponsoring a Merge data analysis and data visualization blog post challenge. Here are all the details you need._
+
+<Divider />
+
+Calling all Ethereans, data scientists, data engineers, data visualizers, developers, and anyone interested in digging into Ethereum data!
+
+The [Merge](https://ethereum.org/en/upgrades/merge/) is coming, providing a more secure and sustainable home for Ethereum -- as well as more data, lots and lots of data. And **the Ethereum community needs your help to make sense of it all.**
+
+- _What new visualizations help provide insight into proof-of-stake Ethereum?_
+- _How do the Consensus-Layer and Execution-Layer interact? Are there differences across client pairs in communication patterns, efficiency, etc? Are their good places to optimise?_
```
Comment (corwintines) [inline src/content/community/grants/merge-data-challenge/index.md:20]:
```suggestion
- _How do the Consensus-Layer and Execution-Layer interact? Are there differences across client pairs in communication patterns, efficiency, etc? Are there good places to optimise?_
```

File: src/content/community/grants/merge-data-challenge/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+1.  Collect and analyze Merge data
+    - Either with existing tools
+    - Or for the extra ambitious, build your own tool and tell us about it!
+2.  Detail out your work in a blog post
+3.  Submit!
+
+<!-- TODO: link to new submission form -->
+
+<ButtonLink to="https://esp.ethereum.foundation/">Submit blog post</ButtonLink>
```
Comment (corwintines) [inline src/content/community/grants/merge-data-challenge/index.md:48]:
```suggestion
<ButtonLink to="https://esp.ethereum.foundation/merge-data-challenge">Submit blog post</ButtonLink>
```

File: src/content/community/grants/merge-data-challenge/index.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+Up to **USD $30,000**
+
+_Entries must be considered sufficiently impactful/insightful by the community judging team to be eligible for rewards._
+
+## Requirements {#requirements}
+
+- Blog post must be in English.
+  - There are no length requirements.
+- Blog posts must be public and original.
+- Data analysis or visualization must concern [Merge](https://ethereum.org/en/upgrades/merge/) network data.
```
Comment (minimalsm) [inline src/content/community/grants/merge-data-challenge/index.md:73]:
```suggestion
- Data analysis or visualization must concern [Merge](/upgrades/merge/) network data.
```

File: src/content/community/grants/merge-data-challenge/index.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+In addition to the prompt questions above, here's a [wishlist of data analysis avenues to explore](https://notes.ethereum.org/@djrtwo/merge-data-comp-wish).
+
+A group of Ethereum community members will evaluate your submissions and awards will go to the top blog posts <Emoji svg text=":trophy:" />
+
+<h2>Deadline <Emoji svg text=":alarm_clock:" /></h2>
+
+The deadline for submissions is **four weeks after the merge** (precise date TBD).
+
+<h2>Prizes <Emoji svg text=":money_with_wings:" /></h2>
```
Comment (minimalsm) [inline src/content/community/grants/merge-data-challenge/index.md:62]:
```suggestion
## Prizes <Emoji text=":money_with_wings:" size={1} /> {#prizes}
```

File: src/content/community/grants/merge-data-challenge/index.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+<ButtonLink to="https://esp.ethereum.foundation/">Submit blog post</ButtonLink>
+
+_You may submit more than one blog post! So long as each posts focuses on a different visualisation or piece of analysis_
+
+In addition to the prompt questions above, here's a [wishlist of data analysis avenues to explore](https://notes.ethereum.org/@djrtwo/merge-data-comp-wish).
+
+A group of Ethereum community members will evaluate your submissions and awards will go to the top blog posts <Emoji svg text=":trophy:" />
+
+<h2>Deadline <Emoji svg text=":alarm_clock:" /></h2>
```
Comment (minimalsm) [inline src/content/community/grants/merge-data-challenge/index.md:58]:
```suggestion
## Deadline <Emoji text=":alarm_clock:" size={1} /> {#deadline}
```

File: src/content/community/grants/merge-data-challenge/index.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+<!-- TODO: link to new submission form -->
+
+<ButtonLink to="https://esp.ethereum.foundation/">Submit blog post</ButtonLink>
+
+_You may submit more than one blog post! So long as each posts focuses on a different visualisation or piece of analysis_
+
+In addition to the prompt questions above, here's a [wishlist of data analysis avenues to explore](https://notes.ethereum.org/@djrtwo/merge-data-comp-wish).
+
+A group of Ethereum community members will evaluate your submissions and awards will go to the top blog posts <Emoji svg text=":trophy:" />
```
Comment (minimalsm) [inline src/content/community/grants/merge-data-challenge/index.md:56]:
```suggestion
A group of Ethereum community members will evaluate your submissions, and awards will go to the top blog posts <Emoji text=":trophy:" size={1} />.
```

File: src/content/community/grants/merge-data-challenge/index.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+  - [Engine API](https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md)
+- [Mega Merge Resource List](https://notes.ethereum.org/@MarioHavel/merge-resources)
+- [Ethereum Roadmap FAQ](https://github.com/timbeiko/eth-roadmap-faq)
+- [Ethereum Energy Consumption](https://ethereum.org/en/energy-consumption/)
+- [Eth R&D](https://discord.gg/VmG7Uxc) _– Discord_
+- [ethstaker](https://invite.gg/ethstaker) _– Discord_
+
+## Support {#support}
+
+For any general support questions about your submission, please email datachallenge@ethereum.org.
```
Comment (minimalsm) [inline src/content/community/grants/merge-data-challenge/index.md:116]:
```suggestion
For any general support questions about your submission, please email [datachallenge@ethereum.org](mailto:datachallenge@ethereum.org).
```

File: src/content/community/grants/merge-data-challenge/index.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+## The challenge {#the-challenge}
+
+**Document your best Merge data insights in the most readable blog post possible – for prizes!**
+
+The Ethereum Foundation is running this challenge because there's a lot to learn and discover from the Merge mainnet activity. Your findings will give the Ethereum community -- from beginners to researchers and client developers -- important insight into the Merge.
+
+Get started now if you want to gather date from both _before_ and _after_ the big event!
+
+ <Emoji svg text=":rocket:" />
```
Comment (minimalsm) [inline src/content/community/grants/merge-data-challenge/index.md:36]:
```suggestion
 <Emoji text=":rocket:" size={1} />
```

File: src/content/community/grants/merge-data-challenge/index.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+_Up until four weeks after the Merge, the Ethereum Foundation is sponsoring a Merge data analysis and data visualization blog post challenge. Here are all the details you need._
+
+<Divider />
+
+![Two people holding hands in field with ethereum logo](./merge-data-challenge.jpg)
+
+Calling all Ethereans, data scientists, data engineers, data visualizers, developers, and anyone interested in digging into Ethereum data!
+
+The [Merge](https://ethereum.org/en/upgrades/merge/) is coming, providing a more secure and sustainable home for Ethereum -- as well as more data, lots and lots of data. And **the Ethereum community needs your help to make sense of it all.**
```
Comment (minimalsm) [inline src/content/community/grants/merge-data-challenge/index.md:19]:
```suggestion
[The Merge](https://ethereum.org/en/upgrades/merge/) is coming, providing a more secure and sustainable home for Ethereum -- as well as more data, lots and lots of data. And **the Ethereum community needs your help to make sense of it all.**
```

File: src/content/community/grants/merge-data-challenge/index.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+sidebarDepth: 2
+---
+
+# The Merge data challenge {#the-merge-data-challenge}
+
+_Up until four weeks after the Merge, the Ethereum Foundation is sponsoring a Merge data analysis and data visualization blog post challenge. Here are all the details you need._
+
+<Divider />
+
+![Two people holding hands in field with ethereum logo](./merge-data-challenge.jpg)
```
Comment (minimalsm) [inline src/content/community/grants/merge-data-challenge/index.md:15]: This is decorative, so it should have an empty alt.

```suggestion
![](./merge-data-challenge.jpg)
```

Comment (minimalsm) [inline src/content/community/grants/merge-data-challenge/index.md:15]: Just noting this feels like an unusual place to have an image. But, it's probably the best solution as we don't have a template to support this (yet).

**Review by minimalsm — CHANGES_REQUESTED** [review]

File: src/content/community/grants/merge-data-challenge/index.md (hunk 10 above)
Comment (Rodrigolvc) [reply src/content/community/grants/merge-data-challenge/index.md:15]: Sorry I missed this. I don't, but tomo might

**Comment by corwintines** [comment]
@Rodrigolvc going to close this out and pull in the link to the grants page once we get that ready to ship.

**Pull request closed without merging**